src/components/DivElement.js

```diff
@@ -1,6 +1,14 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super('div', content);
+  }
+  render() {
+    return super.render();
```

Comment:
If you don't declare your own render function in `DivElement`, it'll automatically call `super.render()` for you without having to redefine lines 8-11 in this function. But you get the idea, nice job 😃